src/features/Chat/ChatMenu.tsx

```diff
@@ -0,0 +1,158 @@
+import styled from 'styled-components';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import BackArrowIcon from '@/static/icons/back-arrow.svg';
+
+import IconButton from '@/components/Buttons/IconButton';
+import { palette } from '@/components/variables';
+import Text from '@/components/Text';
+
+type ChatCategory = 'active' | 'archived' | 'blocked';
+
+const ChatMenu = () => {
+  const { t } = useTranslation('chat');
+  const [showCategories, setShowCategories] = useState(false);
+  const [currentCategory, setCurrentCategory] =
+    useState<ChatCategory>('active');
+  const [chats, setChats] = useState([]);
```

Comment:
also here, i would just use a const for now

Reply:
Fixed!